src/toy.rs

```diff
@@ -4,133 +4,125 @@ extern crate cranelift_simplejit;
 extern crate peg;
 
 use std::mem;
-use std::process;
 
 mod frontend;
 mod jit;
 
 fn main() {
+    run_toy().unwrap();
+}
```

Comment:
Suggestion: rename run_toy to `fn main() -> Result<(), String>` to get this behavior from rustc for "free".